### PR TITLE
Disable beta channels

### PR DESCRIPTION
**Pull request by srbarrios**
## What does this PR change?

This PR aims to improve how we handle actions related to beta channels, depending if we want them enabled or disabled.

### CODE REVIEW

File: salt/server/init.sls [hunk 1]
```diff
@@ -162,8 +162,7 @@ extend_login_timeout:
         - cmd: server_setup
 {% endif %}
 
-# WORKAROUND: 4.4 is needed only until the branching of SUSE Manager 4.4 is completed
-{% if 'head' in grains.get('product_version') or '4.4' in grains.get('product_version') %}
+{% if grains.get('beta_enabled') %}
```
Comment (Bischoff) [inline salt/server/init.sls:165]: rather:
```
{% if 'head' in grains.get('product_version') and  if grains.get('beta_enabled') %}
```
no ?

**Review by Bischoff — CHANGES_REQUESTED** [review]
se suggested change

File: salt/server/init.sls (hunk 1 above)
Comment (srbarrios) [reply salt/server/init.sls:165]: I'm not sure, could we have a case were the product version is different than head, but we still want to change this product_tree_tag to Beta?

For instance, we have a 'uyuni-pr' product_version that we use for those builts coming from branches that we test in our PR test suite. We were working in having the same option but for SUMA branches. In such cases, maybe we want to give more flexibility about the product version we are deploying.

**Review by mcalmer — APPROVED** [review]

File: salt/server/init.sls (hunk 1 above)
Comment (Bischoff) [reply salt/server/init.sls:165]: answered in chat. I'd go for a minimal change and therefore keep the head condition

Comment (srbarrios) [reply salt/server/init.sls]: OK, I applied the suggestion.

**Review by Bischoff — APPROVED** [review]

**Pull request merged**